src/mutation_data.cpp

```diff
@@ -132,7 +132,7 @@ void mutation_branch::load( JsonObject &jsobj )
         int neutral = jo.get_int("neutral", 0);
         int good = jo.get_int("good", 0);
         tripoint protect = tripoint(ignored, neutral, good);
-        new_mut.protection[part_id] = mutation_wet(body_parts[part_id], protect);
+        new_mut.protection[part_id] = mutation_wet(get_body_part_token( part_id ), protect);
```

Comment:
Something generates a lot of errors about bad body parts like "ARMS" and "LEGS" and this one looks like a good candidate.

`map<>::operator[]` creates a new entry if one doesn't exist rather than just crashing. `part_id` seems not to be converted in any way from its json form and in json it is written as "ARMS" or "HANDS", not "ARM_L", like both `body_parts` did expect and `get_body_part_token` does expect.